pkg/controller/constraintstatus/constraintstatus_controller.go

```diff
@@ -109,7 +120,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler, events <-chan event.Generi
 	// Watch for changes to ConstraintStatus
 	err = c.Watch(
 		&source.Kind{Type: &v1beta1.ConstraintPodStatus{}},
-		handler.EnqueueRequestsFromMapFunc(PodStatusToConstraintMapper(util.EventPackerMapFunc())),
+		handler.EnqueueRequestsFromMapFunc(PodStatusToConstraintMapper(false, util.EventPackerMapFunc())),
```

Comment:
why is selfOnly false here?

Reply:
Because the status controller is run by a singleton pod which writes the results of all the statuses into a single constraint.

Reply:
i.e. it is explicitly interested in the statuses of all pods

Reply:
Is it because for the constraint reconciler, we are only interested in the status update events associated with the current pod vs for the constraintStatus reconciler, we are interested in all the status update events across all pods?

Reply:
yep!